cypress/tests/status-card.spec.ts

```diff
@@ -1,7 +1,7 @@
 import { STATUS_CARD_TITLE } from '../constants/tests';
 import { MCGMSCommon } from '../views/mcg-ms-common';
 
-describe('admin dashboard - status card', () => {
+describe('tests for status card of admin dashboard', () => {
```

Comment:
No need to prepend with "tests for" as it's already implicit. See the [example maintained by Cypress Team](https://github.com/cypress-io/cypress-realworld-app/blob/develop/cypress/tests/ui/bankaccounts.spec.ts#L10).